packages/frontend/src/view/components/common/Button.tsx

```diff
@@ -0,0 +1,33 @@
+import cx from 'classnames'
+import React from 'react'
+
+interface ButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {
+  readonly variant?: ButtonVariant
+}
+
+type ButtonVariant = 'primary' | 'outline'
+
+const classNameMap: Record<ButtonVariant, string> = {
+  primary: 'bg-brand text-white',
+  outline: 'bg-transparent border border-brand text-white hover:bg-opacity-20',
+}
```

Comment:
nitpick. I don't see a big benefit over inlining in `cx` `variant === 'primary' && '....'`

Reply:
I would say that if we add new ButtonVariant we would not get any typescript error that we have to add ```variant==='newVariant'``` to cx. With this approach you will get an error that you need to handle the new variant plus the code looks cleaner I would say